ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -506,6 +506,17 @@ tests:
         IPv4=no
       TEST_SUITE: minimal
     workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-ipv4v6
```

Comment:
Just a nit, I think we are now using `e2e-ai` instead of `e2e-metal-assisted`. @celebdor can you confirm ?